app/helpers/operatingsystems_helper.rb

```diff
@@ -13,8 +13,10 @@ def icon(record, opts = {})
     return "" if record.blank? || record.name.blank?
     size = opts[:size] ||= '16x16'
     name = case record.name.downcase
-           when /fedora/
+           when /fedora|fcos|coreos/i
```

Comment:
this is probably not correct, is it?
```suggestion
           when /fedora|fcos/i
```

Reply:
It is, CoreOS is Red Hat's brand and Fedora CoreOS is the only correct name: https://getfedora.org/en/coreos?stream=stable

There is no longer a separate CoreOS project anymore, that is the reason why Flatcar fork exists. So I believe this is correct, you want to see Fedora icon for any "CoreOS" OS name.

Reply:
What's in `/etc/os-release` and how is it identified?